Dockerfile

```diff
@@ -5,6 +5,7 @@ COPY schematics /workspace/schematics/
 COPY package.json package-lock.json /workspace/
 RUN npm i --ignore-scripts
 RUN npm run ngcc
+RUN node node_modules/esbuild/install.js
```

Comment:
@shauke @MaxKless this should actually be part of the `npm postinstall` as simplified in #865

Reply:
Does this mean the other PR needs adaptions once this PR is merged together with the "chore: fix docker build" commit or what is your suggestion?

Reply:
Yes, like that